src/ResourceManager/GuestConfiguration/Commands.GuestConfiguration/AzureRM.GuestConfiguration.psd1

```diff
@@ -0,0 +1,133 @@
+#
+# Module manifest for module 'PSGet_AzureRM.GuestConfiguration'
+#
+# Generated by: Microsoft Corporation
+#
+# Generated on: 8/8/2018
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+# RootModule = ''
+
+# Version number of this module.
+ModuleVersion = '0.9.0-preview'
+
+# Supported PSEditions
+# CompatiblePSEditions = @()
+
+# ID used to uniquely identify this module
+GUID = 'db510f7c-49ef-4eb6-b30e-d7a80fdb7ed4'
+
+# Author of this module
+Author = 'Microsoft Corporation'
+
+# Company or vendor of this module
+CompanyName = 'Microsoft Corporation'
+
+# Copyright statement for this module
+Copyright = 'Microsoft Corporation. All rights reserved.'
+
+# Description of the functionality provided by this module
+Description = 'Microsoft Azure PowerShell - Azure Guest Configuration Policy cmdlets. Allows querying guest configuration policy assignments, assignment reports and compliance statuses.'
```

Comment:
We could end this line with "status" rather than "statuses"

Reply:
done